vocabularies/UI.xml

```diff
@@ -1389,6 +1390,59 @@ The trend is
         </Member>
       </EnumType>
 
+      <Term Name="Note" Type="UI.NoteType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Visualization of a note attached to an entity" />
+        <Annotation Term="Core.LongDescription">
+          <String>Administrative data is given by the annotations
+          [`Common.CreatedBy`](Common.md#CreatedBy),
+          [`Common.CreatedAt`](Common.md#CreatedAt),
+          [`Common.ChangedBy`](Common.md#ChangedBy),
+          [`Common.ChangedAt`](Common.md#ChangedAt)
+          on the same entity type.</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="NoteType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="Title" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.Description" String="Title of the note" />
+          <Annotation Term="Core.LongDescription">
+            <String>The title of a note is hidden with an annotation `@UI.Note/Title/@UI.Hidden`, not with
+            an annotation on the property targeted by `@UI.Note/Title`.</String>
+          </Annotation>
+          <Annotation Term="Validation.ApplicableTerms">
+            <Collection>
+              <String>UI.Hidden</String>
+            </Collection>
+          </Annotation>
+        </Property>
+        <Property Name="Content" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Content of the note, as a string" />
+          <Annotation Term="Core.LongDescription">
+            <String>The property targeted by `@UI.Note/Content` must be annotated with `Core.MediaType` and
+            `Common.SAPObjectNodeTypeReference`. When it is tagged with `Core.IsLanguageDependent`,
+            another property of the same entity type that is tagged with [`Common.IsLanguageIdentifier`](Common.md#IsLanguageIdentifier)
+            determines the language of the note.</String>
+          </Annotation>
+          <Annotation Term="Validation.ApplicableTerms">
+            <Collection>
+              <String>Core.MediaType</String>
+              <String>Core.IsLanguageDependent</String>
+              <String>Common.SAPObjectNodeTypeReference</String>
+            </Collection>
+          </Annotation>
+        </Property>
+        <Property Name="Type" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="A type used for grouping notes" />
+        </Property>
+        <Property Name="MaximalLength" Type="Edm.Int32" Nullable="true">
+          <Annotation Term="Core.Description" String="Type-specific maximal length of the content of the note" />
+        </Property>
+        <Property Name="MultipleNotes" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Whether the type allows multiple notes for one object" />
```

Comment:
This description left me with the question: which type - entity type, or the Type property in UI.Note?

Reply:
Sounds good - and as I claimed that (at least imo) the note type is not mandatory - can I expect that this property is per entity type instance?